core/src/main/scala/org/apache/spark/ui/storage/StoragePage.scala

```diff
@@ -49,6 +49,7 @@ private[ui] class StoragePage(parent: StorageTab) extends WebUIPage("") {
 
   /** Header fields for the RDD table */
   private val rddHeader = Seq(
+    "RDD ID",
```

Comment:
I see, the idea is that the ID is exposed elsewhere in the output, so is useful to expose? makes sense. You could call the column just "ID" to let it be narrower.

Reply:
Thanks @srowen .I will update right now.